content/en/getting_started/serverless/_index.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Getting Started with Serverless Monitoring
+kind: documentation
+
+further_reading:
+    - link: '/agent/basic_agent_usage/'
+      tag: 'Documentation'
+      text: 'Basic Agent Usage'
+---
+
+## Overview
+
+_Serverless_ is a model where developers build and run applications and services using a cloud provider, rather than managing infrastructure themselves. Datadog [Serverless Monitoring][10] collects metrics, logs, and traces from your serverless infrastructure, enabling you to monitor your application's health and performance.
+
+{{< img src="getting_started/serverless/serverless_view.png" alt="Serverless Monitoring: Serverless View, an explorer page" style="width:80%;">}}
+
+This guide makes use of a serverless [sample app][1] that you can launch with one click. This app has Serverless Monitoring preconfigured. Follow this guide to see how you might troubleshoot a problem in your sample app, and what kinds of visibility Serverless Monitoring can provide.
+
+### Install the sample app
+
+1. [Launch the CloudFormation Stack][9]. This link brings you to a **Create stack** page in CloudFormation.
+2. Enter your Datadog API key and Datadog site ({{< region-param key="dd_site" code="true" >}}). 
+
+  {{< img src="getting_started/serverless/aws_create_stack.png" alt="Close-up of two functions" style="width:80%;">}}
+
+  Then, acknowledge IAM capabilities and click **Create Stack**.
+
+3. After the stack has been created, open the Outputs tab.
+
+  {{< img src="getting_started/serverless/aws_outputs.png" alt="Close-up of two functions" style="width:80%;">}}
+
+  Invoke your sample app a few times by visiting the `ApiGatewayInvokeURL`. This returns a "Sent message to SNS" success message.
+
+Each invocation executes the following:
+
+```python
+import boto3, os
+
+def handler(event, context):
+    sns = boto3.client('sns')
+
+    sns.publish(
+        TopicArn=os.environ.get("SNS_TOPIC_ARN"),
+        Message='Message sent to SNS'
+        )
+
+    return {
+        "body": "Sent message to SNS",
+        "statusCode": 200
+    }
+```
+
+You can [see your sample app functions in Serverless View][11].
+
+{{< img src="getting_started/serverless/dd_serverless_view.png" alt="Serverless Monitoring: Serverless View, an explorer page" style="width:80%;">}}
```

Comment:
```suggestion
    Each invocation executes the following:

    ```python
    import boto3, os

    def handler(event, context):
        sns = boto3.client('sns')

        sns.publish(
            TopicArn=os.environ.get("SNS_TOPIC_ARN"),
            Message='Message sent to SNS'
            )

        return {
            "body": "Sent message to SNS",
            "statusCode": 200
        }
    ```

    You can [see your sample app functions in Serverless View][11].

    {{< img src="getting_started/serverless/dd_serverless_view.png" alt="Serverless Monitoring: Serverless View, an explorer page" style="width:80%;">}}
```

make sure this suggestion works in the preview